src/erlcloud_guardduty.erl

```diff
@@ -123,11 +123,7 @@ guardduty_request_no_update(Config, Method, Path, Body, QParam) ->
     end.
 
 encode_body(undefined) ->
-    <<>>;
```

Comment:
This removal seems iffy. Why?

Reply:
It's explained in commit eb0e45a's message:

> `encode_body/1`
> is only ever called locally via
> `guardduty_request_no_update/5`
> 
> `guardduty_request_no_update/5`
> is only ever called locally via
> `guardduty_request/5`
> 
> `guardduty_request/5`
> is only ever called locally with
> `Body =:= undefined`
> 
> (maybe it was intended, initially, to cover more use cases, but undefined ended up being the only one present)

It also took me a while to find out why `dialyzer` was complaining.

Reply:
Interesting. I guess it probably was planned for some future use. I guess that's fine with me.